server/controllers/documentsControllers.js

```diff
@@ -103,12 +103,21 @@ export default {
         order: '"createdAt" DESC'
       })
         .then((documents) => {
-          const settings = limit && offset ? {
-            totalCount: documents.count,
-            pages: Math.ceil(documents.count / limit),
-            currentPage: Math.floor(offset / limit) + 1,
-            pageSize: documents.rows.length
-          } : null;
+          // const settings = limit && offset ? {
+          //   totalCount: documents.count,
+          //   pages: Math.ceil(documents.count / limit),
+          //   currentPage: Math.floor(offset / limit) + 1,
+          //   pageSize: documents.rows.length
+          // } : null;
+          let settings;
+          if (limit && offset) {
+            settings = {
```

Comment:
Yes, I did. I think you are looking at the wrong branch. https://github.com/andela-aalabi/DMS-papyrus/blob/feedback/server/controllers/documentsControllers.js